### ISSUE TITLE
when I press start it starts fine but when I press any button in the control it says index out of bounds of array :/

### ISSUE THREAD

**Issue by piikav** [Axadiw/MFIGamepadFeeder#76]

**Comment by rubikstriangle** [comment]
Yeah, that's what I get as well.  I'm using a numbus+ which I assume is the issue.  What are  you using?

**Comment by BierJess** [comment]
For others who get this issue, this worked for me when getting the index out of bounds error:

1. Install the x86 installer even if your windows is x64. For some reason my x64 windows didn't work with the installer, but the x86 one did. 
2. Find the new config files for the nimbus+ on here -- in the comments at the link below, user nnsense saved it as a txt file and you just have to change the extension and move it to the config folder, using their instructions. The link to the discussion is: https://github.com/Axadiw/MFIGamepadFeeder/pull/73
